rc/base/ctags.kak

```diff
@@ -19,9 +19,9 @@ def -params ..1 \
             fi
             cat "$namecache"
         done' \
-    -docstring %{tag [<symbol>]: jump to a symbol's definition
+    -docstring %{ctags-search [<symbol>]: jump to a symbol's definition
 If no symbol is passed then the current selection is used as symbol name} \
-    tag \
+    ctags-search \
```

Comment:
@mawww If I rename the command to `ctags-jump`, will it be enough proof that the command needs to jump into the `jump` client :p ?